### PR TITLE
GATE-2043: Uses teams rule actions enum defined in cloudflare-go

### PR DESCRIPTION
**Pull request by Michael9127**
Just pinning commit until new cloudflare-go version is released.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,7 +3,7 @@ module github.com/cloudflare/terraform-provider-cloudflare
 go 1.15
 
 require (
-	github.com/cloudflare/cloudflare-go v0.21.0
+	github.com/cloudflare/cloudflare-go v0.21.1-0.20210905215638-8d67feea1cc0
```
Comment (jacobbednarz) [inline go.mod:6]: can you please pull this one out? you/i can shim it in locally if we need to confirm it running. we have automated steps for bumping releases and these files will conflict with that.

Comment (Michael9127) [reply go.mod]: theres no rush on this, so im fine with it not building for now until the next release. Just wanted to push it up while I was cleaning out the RM.

**Comment by jacobbednarz** [comment]
Depends on cloudflare/cloudflare-go#702

File: go.mod (hunk 1 above)
Comment (jacobbednarz) [reply go.mod]: 🙇‍♂️

**Pull request merged**